whelk-core/src/main/resources/ext/marcframe.md

```diff
@@ -0,0 +1,203 @@
+
+The most common reason for a marc-prefix is LC combining two fields so it is not possible to separate the data making a reconversion impossible, or that they haven't made any attempts to code the information either with nac - no attempt to code, or ignore for things which have no meaning outside MARC21.
+
+There are also certain postprocessing steps which can handle particular fields, for example date/year which needs to be exported to several fields (E.g. 008 + 264) depending on how they are formatted.
+
+## Syntax in MARCFRAME
+
+This is a list of keys in Marcframe which control certain functionality like repetition, grouping, inclusion of patterns etc. 
+
+`about` enables shortforms of more complicated or nested entitites.
+
+`aboutAlias` used to include an about form.
+
+`aboutEntity` Defines which entity the property is a part of, **thing** means instance in bibliographic data.
+
+`aboutType` -
+
+`addLink` Is used when the entities (classes) are repeatable. If the entity can only occur once use `link`. The repetition of MARC21 fields is stated either in Libris Formathandbok or LC MARC21 specification.
+
+`addProperty`(R) and `property` (NR) is used to define whether a property is repeatable or not.
+
+`embedded` could be set to *true* which means that the entity is a so called structured value and it's value is always embedded locally to the graph and not linked to. In RDF also known as a b-node. The most common types is titles, identifiers and notes.
+
+`include` includes the pattern which match the value.
+
+`inherit` creates an inheritance from the specified field.
+
+`link` see addLink.
+
+`match` used for matching values in indicator or occurence of fields with a `when` construction. [See exemple 2, which illustrates matching the right type of agent or work in the **$100** field]
+
+`property` see addProperty.
+
+`punctuationChars` states the interpunctuation which is stripped away and restored to the string/subfield in the conversion.
+
+`splitValuePattern` the regexp pattern used to separate strings into properties. [Example: identifier with trailing qualifier].
+
+`splitValueProperties` defines the properties which the separated strings groups should map to. [Example: value, qualifier].
+
+ `pendingResource` makes it possible to create an hierarchical structure of entities.
+
+`required: true` is used for the conversion to MARC21 to know if a field should be conversed if certain subfields are missing. It could also be reversed with `supplementary: true` which tells that the occurence of certain fields is not enough to convert it with a full meaning. [Example: `$a` is often required, `$2` is often supplementary].
```

Comment:
s/to know/to indicate/ ?

s/conversed/converted/ ?

It's not clear what happens when you add `supplementary: true` to a field with `required: true`. The way it's written now gives me the impression that `supplementary` overrides `required`.

Reply:
I rewrote this section. I agree it was not clear about the intended usage